examples/common/pigweed/rpc_services/Device.h

```diff
@@ -481,6 +482,22 @@ class Device : public pw_rpc::nanopb::Device::Service<Device>
         return pw::OkStatus();
     }
 
+    virtual pw::Status TriggerIcdCheckin(const pw_protobuf_Empty & request, pw_protobuf_Empty & response)
```

Comment:
I'm not sure i understand the point of this function. It seems to overrule the logic in the ICDManager to send a check-in message at the end of the IdleModeDuration.

I don't think this should be implemented. Comment applies to the input argument to set the timer value.